source/xor.js

```diff
@@ -0,0 +1,26 @@
+import _curry2 from './internal/_curry2';
+
+/**
+ * Exclusive disjunction logical operation.
+ * Returns `true` if one of the arguments is truthy and the other is falsy.
+ * Otherwise, it returns `false`.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @sig a -> b -> Boolean
+ * @param {Any} a
+ * @param {Any} b
+ * @return {Boolean} true if one of the arguments is truthy and the other is falsy
+ * @see R.or, R.and
+ * @example
+ *
+ *      R.xor(true, true); //=> false
+ *      R.xor(true, false); //=> true
+ *      R.xor(false, true); //=> true
+ *      R.xor(false, false); //=> false
+ */
+var xor = _curry2(function xor(a, b) {
+  return Boolean(a || b) && !(a && b);
```

Comment:
How about a simpler expression like `Boolean(!a ^ !b)`?

Reply:
I like it. 
I simply didn't knew about the existence of the xor bitwise operator. I agree that your solution is simpler, so I'm changing it.

Reply:
I'm not sure I like this so well.  The multiple levels of coercion seem a bit much: coerce both parameters to booleans, invert both, coerce the booleans to Numbers, apply the bitwise operator, then coerce the result again to a boolean.  I think I prefer the explicit approach above.

Reply:
The `^`-operator using version seems to be faster on both Node 8.11.3 and 10.8.0.  Of course, once you add currying, performance drops by a factor of 10 and so does the performance advantage of the `^`-operator using version.

Reply:
@polytypic so, I guess now the debate is between which solution is simple/easier to read, and also which performs better. Did you use any profiling tool for the comparison in performance? If so, could you please show me the results? 
Thanks!

Reply:
I used Benchmark.js.  Here is the biggest diff (not curried):

```
      95,043/s    1.00x  vs1000.reduce(xorBitwise)
      70,951/s    1.34x  vs1000.reduce(xorLogical)
```

`vs1000` is an array of 1000 elements (of number `1`, IIRC).  In all cases that I tried the version using `^` (named `xorBitwise` above) was faster.  Once you add Ramda's `curry`, the difference gets much smaller:

```
       9,532/s    1.00x  vs1000.reduce(xorBitwise)
       8,924/s    1.07x  vs1000.reduce(xorLogical)
```

Adding `R.curry` has a cost of one decimal order of magnitude in this case.

Reply:
I don't have a major objection to the bitwise version.  But there is something odd about using it for Booleans.